odoo/addons/base/models/ir_translation.py

```diff
@@ -157,7 +157,7 @@ class IrTranslation(models.Model):
     _log_access = False
 
     name = fields.Char(string='Translated field', required=True)
-    res_id = fields.Integer(string='Record ID', index=True)
+    res_id = fields.Integer(string='Record ID')
```

Comment:
It does seem to be useful in some cases. Some usage statistics in a real database:

```
                 index                  |     table      |  idx_scan  | pg_size_pretty 
----------------------------------------+----------------+------------+----------------
 ir_translation_unique                  | ir_translation |  344815142 | 336 MB
 ir_translation_ltn                     | ir_translation |      96276 | 209 MB
 ir_translation_name_lang_res_id_id_idx | ir_translation | 1524412718 | 192 MB
 ir_translation_type_name_res_id_idx    | ir_translation |   87583075 | 179 MB
 ir_translation_module_index            | ir_translation |  243636669 | 167 MB
 ir_translation_src_md5                 | ir_translation |   12213626 | 164 MB
 ir_translation_type_index              | ir_translation |     151163 | 162 MB
 ir_translation_comments_pattern_idx    | ir_translation |          0 | 156 MB
 ir_translation_comments_index          | ir_translation |          0 | 156 MB
 ir_translation_pkey                    | ir_translation | 4755952486 | 134 MB
 ir_translation_res_id_index            | ir_translation |  294367430 | 90 MB
 ir_translation_model_unique            | ir_translation |     111931 | 85 MB
 ir_translation_code_unique             | ir_translation |     129030 | 7480 kB
```

We can however see that the index on `comments` is unused, even though we search on this column in `get_translations_for_webclient()`.